src/Notifly/Exception/MessageTypeUnsupported.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Notifly\Exception;
+
+class MessageTypeUnsupported extends \Exception
```

Comment:
Can you prefix this class name with `Notifly`
i.e. `NotiflyMessageTypeUnsupported`